src/components/boards/Board.tsx

```diff
@@ -324,6 +327,17 @@ function Board({
             )}
           </ActionIcon>
         </Tooltip>
+        {!eraseDrawablesOnClick && (
+          <Tooltip label="Clear Drawings">
+            <ActionIcon
+              variant={editingMode ? "filled" : "default"}
+              size="lg"
+              onClick={() => setShapes([])}
```

Comment:
Instead of `setShapes`, there's already a `clearShapes` function in store.ts which also sets the dirty flag.